main/src/ca/uwaterloo/flix/language/ast/Scheme.scala

```diff
@@ -99,7 +99,7 @@ object Scheme {
   def lessThanEqual(sc1: Scheme, sc2: Scheme, classEnv: Map[Symbol.ClassSym, Ast.ClassContext])(implicit flix: Flix): Boolean = {
     checkLessThanEqual(sc1, sc2, classEnv) match {
       case Validation.Success(_) => true
-      case Validation.Failure(_) => false
+      case _failure => false
```

Comment:
Now that we use this variable pattern `case _failure` I've adopted the Flix style of using an underscore as prefix to signal that the wildcard match is not used while keeping it easy to read. Is this the right way to do it or should we just use `case _ => ???` instead or is there a better Scala-specific way of doing things?

Reply:
I think this is ok.